nzsl-dictionary-scripts/env-production/main.tf

```
@@ -0,0 +1,71 @@
+
+terraform {
```

nitpick: unneeded whitespace:

```suggestion
terraform {
```